script.js

```diff
@@ -0,0 +1,124 @@
+const {
+  speakerName = [
+    'Yochai Benkler',
+    'SohYeong Noh',
+    'Churchil Okonkwo',
+    'Lila tetrikov',
+    'Kilnam Chom',
+    'Chimamanda Adichie',
+  ],
+  photoImage = [
+    'images/yochai-benkler.jpg',
+    'images/sohyang.jpg',
+    'images/churchil-okonkwo.jpg',
+    'images/lila-tretikov.jpg',
+    'images/kilnam-chom.jpg',
+    'images/chimamanda.jpg',
+  ],
+  speakerProf = [
+    'Berman Professor of Entrepreneurial Legal Studies at Havard Law School',
+    'Director of Art Center Nabi and a board member of CC Korea',
+    'As an Assistant Program Manager at DC Water',
+    'Deputy CTO at Microsoft.',
+    'PhD in Computer Science, UCLA, 1974 ·',
+    'Novelist, short story writer, non-fiction writer',
+  ],
+  aboutSpeaker = [
+
+    "Benkler studies commons-based peer production, and published his seminal book 'The Wealth of Networks in 2006.'",
+    'As the main venue for new media art production in korea, nabi promotes cross-disciplinary collaboration and understanding among science technology, humanites and the arts',
+    'A Water Services Specialist at District of Columbia Water and Sewer Authority, Washington DC, United State of America.',
+    'A strategic, customer-centric innovator, known for rapid growth. Experience includes creating and scaling companies, operating a top-5 web and media property,',
+    "Professor Chon contributed to the Internet's growth in Asia through his extensive work in advancing Internet initiatives, research, and development.",
+    'Chimamanda Ngozi Adichie is a Nigerian writer whose works include novels, short stories and nonfiction',
+  ],
+
+} = ['speakers'];
```

Comment:
[OPTIONAL] note: I have only made this optional because you have done an incredible job on your project. Refactoring the structure of you hardcoded data would mean rewriting your entire script which may cause a lot of unexpected bugs. However, this is part of the project requirements and I highly suggest that you at least try to implement it and make sure that you follow these guidelines in the upcoming projects. This will contribute to your growth and teach you how to write better scripts in your software development career.

In the `script.js` file.

- The list of speakers should contain a speaker object for each that holds their specific data. e.g
```
const speakerList = [ { name: "Speaker Name",... } ]
```

- The object should contain all the information needed to generate the dynamic element, at least the name, role, description, and image URL of the speaker. For example:
```
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
```
You currently have the information for each speaker saved in different arrays and that is not the best practice in saving data.